mockwebserver/src/main/java/okhttp3/mockwebserver/MockWebServer.java

```diff
@@ -449,6 +449,7 @@ public void processConnection() throws Exception {
               .protocol(protocol)
               .listener(framedSocketListener)
               .build();
+          framedConnection.start();
```

Comment:
Oh, right. I assume it's no big deal, but should we create an issue to enforce it?